<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A2HUG71" ignored="0" name="testing" tests="7" failures="1" timestamp="08 Sep 2019 17:16:47 GMT" time="154.877" errors="0">
  <testcase name="loginGmail" time="10.956" classname="tests.GmailLoginSuccess"/>
  <testcase name="searchEmailInInbox" time="6.083" classname="tests.SearchEmailInInbox">
    <skipped/>
  </testcase> <!-- searchEmailInInbox -->
  <testcase name="searchEmailInInbox" time="10.762" classname="tests.SearchEmailInInbox"/>
  <testcase name="loginFails" time="10.482" classname="tests.GmailLoginFails"/>
  <testcase name="retryFailedTest" time="9.441" classname="tests.RetryFailedTest">
    <skipped/>
  </testcase> <!-- retryFailedTest -->
  <testcase name="retryFailedTest" time="8.4" classname="tests.RetryFailedTest">
    <skipped/>
  </testcase> <!-- retryFailedTest -->
  <testcase name="retryFailedTest" time="8.017" classname="tests.RetryFailedTest">
    <failure type="java.lang.AssertionError" message="Wrong Password error message is displayed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Wrong Password error message is displayed expected [false] but found [true]
at tests.RetryFailedTest.retryFailedTest(RetryFailedTest.java:37)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- retryFailedTest -->
</testsuite> <!-- testing -->
